usermods/audioreactive/audio_source.h

```diff
@@ -15,7 +15,7 @@
 #else
 #define SRate_t int
 #endif
-#include "ES8388.h"
+// #include "ES8388.h"
```

Comment:
Remove entirely rather than comment and also remove from platformio.ini

Reply:
Fixed.